src/test/java/com/es/phoneshop/model/product/ArrayListProductDaoTest.java

```diff
@@ -16,6 +23,35 @@ public void setup() {
 
```

Comment:
where are the tests when NoSuchElementException can be thrown? While delete, getProduct